src/konnektor/network_planners/generators/redundant_minimal_spanning_tree_network_generator.py

```diff
@@ -22,31 +24,31 @@ def __init__(
         _initial_edge_lister: NetworkGenerator = None,
     ):
         """
-        The `RedundantMinimalSpanningTreeNetworkGenerator` is an approach that tries to increase
-        robustness to `Transformation` failures in an MST-like network.
+        The ``RedundantMinimalSpanningTreeNetworkGenerator`` is an approach that tries to increase
+        robustness to ``Transformation`` failures in an MST-like network.
 
-        This algorithm executes the MST algorithm `n_redundancy` times, always removing
-        already-selected `Transformations` in each iteration, and finally builds the overlay of all the networks.
+        This algorithm executes the MST algorithm ``n_redundancy`` times, always removing
+        already-selected ``Transformations`` in each iteration, and finally builds the overlay of all the networks.
```

Comment:
Should a comment reflecting the new behaviour be included in the docstring?

Reply:
yes, thanks for catching that!